### PR TITLE
Task-11-profile-courses-ui

### PR DESCRIPTION
**Pull request by sergevei**
Task-11-profile-courses-ui

### CODE REVIEW

File: src/components/app.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
 import React from 'react';
+import Profile from '../containers/profile/profile.jsx';
```
Comment (OlegPanasyuk) [inline src/components/app.js]: `import Profile from '../containers/profile/profile.jsx';`

Instead, use the template

`import Profile from '../containers/profile';`

And rename `profile.jsx` into `index.jsx`

File: src/containers/profile/profileTab/profileTab.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import styles from './styles.module.scss';
+import { FaFacebookF, FaLinkedinIn, FaTwitter} from "react-icons/fa";
+
+export default function profileTab() {
+  const [update, setUpdate] = useState(true);
+
+  return ( 
+    <section className={styles.profile}>
```
Comment (OlegPanasyuk) [inline src/containers/profile/profileTab/profileTab.jsx]: Use functionality of ES6 like a destructing.

`const {className1, className2}  = styles;`

File: src/containers/profile/profile.jsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import styles from './styles.module.scss';
+import { FaWindowClose, FaUserCircle } from 'react-icons/fa';
+
+import ProfileTab from './profileTab/profileTab.jsx';
```
Comment (melonholy) [inline src/containers/profile/profile.jsx]: use .js file type instead of .jsx

File: src/containers/profile/profile.jsx [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        <button className={styles.active} onClick={() => setComponent(0)}>
+          Profile
+        </button>
+        <button onClick={() => setComponent(1)}>Courses</button>
+        <button onClick={() => setComponent(2)}>Account</button>
+        <button onClick={() => setComponent(3)}>Settings</button>
+      </section>
+      <section className={styles.main}>
+        {getComponent === 0 && <ProfileTab />}
+        {getComponent === 1 && (
```
Comment (OlegPanasyuk) [inline src/containers/profile/profile.jsx]: In future, it needs to change it to Route

File: src/containers/profile/profileTab/profileTab.jsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      <section className={styles.user_fio}>
+        <input type="text" placeholder="First name" disabled={update} defaultValue='John'/>
+        <input type="text" placeholder="Last name" disabled={update} defaultValue='Doe'/>
+      </section>
+      <section className={styles.description}>
+        <input type="text" placeholder="Description" disabled={update} />
+      </section>
+      <section className={styles.social}>
+        <section>
+          <span><FaTwitter style={{ color:'#1da1f2 '}}/>http://twitter.com/</span>
```
Comment (melonholy) [inline src/containers/profile/profileTab/profileTab.jsx]: use const colors from colors.scss

File: src/containers/profile/profileTab/styles.module.scss [hunk 6]
```diff
@@ -0,0 +1,144 @@
+@import '../../../scss/shared/breakpoints.scss';
```
Comment (melonholy) [inline src/containers/profile/profileTab/styles.module.scss]: Duplicated code, try to see how to use @media query from another repo for instance

File: src/containers/profile/profile.jsx [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+          <section>
+            <p>Settings</p>
+          </section>
+        )}
+      </section>
+    </section>
+  );
+}
+
+export default profile;
```
Comment (melonholy) [inline src/containers/profile/profile.jsx]: change export default profile to export const Profile = () =>{
}  
and than use import { Profile } from '../containers/profile'; for importing component
also you need to name components with Upper letter

**Pull request merged**